config/projects.yml

```diff
@@ -118,7 +119,6 @@ block-bundle:
       target_php: '7.3'
       versions:
         symfony: ['4.4']
-        sonata_admin: ['3']
```

Comment:
on 3.x is a dev requirement, on 4.x it is not a requirement

Reply:
I think these checks should be added only for requirement, not dev-requirement.

If it's a dev-requirement, it will still be loaded on every build by the way.